var/spack/repos/builtin/packages/hpcc/readme.md

```diff
@@ -0,0 +1,27 @@
+
+Command line examples for HPCC installation with Spack:
```

Comment:
I'm not sure if this readme is necessary, these are just generic install commands that apply to every package.

Reply:
I personally like to have a small cheat-sheet, I am certainly missing it for other packages. At least it shows examples and tested combination of dependencies. Still I can delete it.

Reply:
On the other hand it is really redundant, I have deleted it.